sdk/src/main/AndroidManifest.xml

```diff
@@ -26,7 +26,6 @@
             android:permission="android.permission.BIND_JOB_SERVICE"
             android:exported="true" />
         <service android:name=".RadarForegroundService"
-            android:foregroundServiceType="location"
-            android:stopWithTask="true" />
```

Comment:
This flag ties the Service to the Activities for the app, such that if the user kills the app, the fg service will be killed. However - @jsani-radar and I have observed some different behavior here, so it may be different depending on which Android version is used.